lib/improver/nbhood/circular_kernel.py

```diff
@@ -170,10 +170,11 @@ def apply_circular_kernel(self, cube, ranges):
         self.kernel = circular_kernel(fullranges, ranges,
                                       self.weighted_mode)
         # Smooth the data by applying the kernel.
-        if self.sum_or_fraction is "fraction":
-            total_area = np.sum(self.kernel)
-        elif self.sum_or_fraction is "sum":
+        if self.sum_or_fraction == "sum":
             total_area = 1.0
+        else:
+            # sum_or_fraction is in fraction mode
+            total_area = np.sum(self.kernel)
```

Comment:
When it's bi-variate (a default case and another string) it'd be neater to have 'fraction_mode' or similar as a bool flag, avoids having to do string comp and arg checking above.